website/public/openapi/openapi.0.1.0.yaml

```diff
@@ -13,9 +13,7 @@ tags:
   - name: Competitions
     description: Endpoints related to competitions, which are distinct application processes for the same funding opportunity
   - name: Applications
-    description: Endpoints related to submitting applications for a given competition
-  - name: Application Reviews
-    description: Endpoints related to reviewing applications for a given competition
```

Comment:
I think these changes got added because you might not have re-run `npm install` to pick up the latest `@common-grants/core` library (the pre-release)

The same is true for all other changes in the `/website/public/` folder